packages/editor-ui/src/api/helpers.ts

```diff
@@ -93,3 +93,7 @@ export async function makeRestApiRequest(context: IRestApiContext, method: Metho
 export async function get(baseURL: string, endpoint: string, params?: IDataObject, headers?: IDataObject) {
 	return await request({method: 'GET', baseURL, endpoint, headers, data: params});
 }
+
+export async function post(baseURL: string, endpoint: string, params?: IDataObject, headers?: IDataObject) {
```

Comment:
@mutdmour I created post helper function because I will use it in this branch, and in the task for contact propmt.